src/DotNetty.Common/Utilities/SystemPropertyUtil.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+
+namespace DotNetty.Common.Utilities
+{
+    /// <summary>
+    /// A collection of utility methods to retrieve and parse the values of the .Net Environment properties.
+    /// </summary>
+    public static class SystemPropertyUtil
+    {
+        public static bool Contains(string key)
+        {
+            return Get(key) != null;
+        }
+
+        public static string Get(string key)
+        {
+            return Get(key, null);
+        }
+
+        public static string Get(string key, string def)
+        {
+            if (key == null)
+            {
+                throw new ArgumentNullException("key");
+            }
+            if (string.IsNullOrWhiteSpace(key))
+            {
+                throw new ArgumentException("key must be non empty");
+            }
+            string value = null;
+            try
+            {
+                value = Environment.GetEnvironmentVariable(key);
+            }
+            catch
+            {
+                // ignored
+            }
+
+            if (value == null)
+            {
+                return def;
+            }
+            return value;
+        }
+
+        public static int GetInt(string key, int def)
+        {
+            string value = Get(key);
+            if (value == null)
+            {
+                return def;
+            }
+            value = value.Trim().ToLower();
```

Comment:
leave it be for now. I have an upcoming PR fixing minor things like this in `SystemPropertyUtil`